ts/src/bigone.ts

```diff
@@ -1185,14 +1185,15 @@ export default class bigone extends Exchange {
          * @description fetch the deposit address for a currency associated with this account
          * @param {string} code unified currency code
          * @param {object} params extra parameters specific to the bigone api endpoint
-         * @returns {object} an [address structure]{@link https://docs.ccxt.com/#/?id=address-structure}
+         * @returns {object} an [address structure]{@link https://docs.ccxt.com/en/latest/manual.html#address-structure}
          */
         await this.loadMarkets ();
         const currency = this.currency (code);
         const request = {
             'asset_symbol': currency['id'],
         };
-        const response = await this.privateGetAssetsAssetSymbolAddress (this.extend (request, params));
+        const [ networkCode, paramsOmitted ] = this.handleNetworkCodeAndParams (params);
```

Comment:
shouldn't this be working?
```
 n bigone fetchDepositAddress "USDT" '{"network":"TRC20"}'
Debugger attached.
2023-06-23T14:50:54.311Z
Node.js: v18.14.0
CCXT v3.1.48
bigone.fetchDepositAddress (USDT, [object Object])
NotSupported bigone - TRC20 network was not found for USDT, use one of EOS, Bitcoin, Ethereum, BinanceSmartChain, Tron
```

Reply:
@carlosmiei at first  this needs to be merged - https://github.com/ccxt/ccxt/pull/18299
 then this current PR should work